Paws4Me/Paws4Me/Features/Sign Up User/View Model/Model/User.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  User.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/05/19.
+//
+
+import Foundation
```

Comment:
Think we can remove this empty file?

Reply:
I agree. Apologies that I missed this 👍🏻